fastboot-tests/fixtures/treeshake/ember-cli-build.js

```diff
@@ -0,0 +1,11 @@
+const EmberApp = require('ember-cli/lib/broccoli/ember-app');
+
+module.exports = function(defaults) {
+  let app = new EmberApp(defaults, {
+    'ember-bootstrap': {
+      whitelist: ['bs-button-group']
```

Comment:
Seems like we should add to the blueprint, as well, at least to create empty arrays. Is there a `blacklist` config, as well? It would require adding array value support to ember-cli-build-config-editor.